api/app/Http/Controllers/PostController.php

```diff
@@ -46,13 +56,31 @@ public function store(Request $request)
         
         //store in the db
         $post = new Post;
+
         $post->name = $request->name;
         $post->content = $request->content;
         $post->is_job = $request->input('job_form') ? 1 : 0;
+
+        if ($post->is_job){
+            $post->form_id = $request->input('job_form');
+        }
+        $post->user_id = auth()->user()->id;
         $post->save();
+        $this -> sendEmail($post);
```

Comment:
Is the email supposed to be sent on every new post or every new job post? Because now it sends it on every post, even thought it's not a job post. But the e-mail specifically mentions "New Job Post"

Reply:
Whoopsies, thanks for catching!